src/js/common/schemaform/FormPage.jsx

```diff
@@ -17,6 +18,15 @@ function focusForm() {
   }
 }
 
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', window.VetsGov.scroll || {
```

Comment:
Yeah. I didn't realize it was still used for things. It used to be used to swap urls out for e2e tests, but I think we replaced that with the environment.js file.